distribution/docs/src/main/jdocs/content/_plugins/pre-authorization-plugin-intro.adoc

```diff
@@ -0,0 +1,14 @@
+:type: pluginIntro
+:status: published
+:title: Pre-Authorization Plugins
+:link: _pre_authorization_plugins
+:summary: Perform any changes needed before security rules are applied.
+:plugintypes: preauthorization
+:order: 01
+
+Pre-delivery plugins are invoked before any security rules are applied. 
```

Comment:
I think it should be Pre-Delivery when part of the name of a specific plugin, and lowercase when describing pre-delivery plugins in general.